demo/src/androidTest/java/com/novoda/movies/topten/RateableMovieViewModelFixtures.java

```diff
@@ -1,99 +0,0 @@
-package com.novoda.movies.topten;
-
-import android.support.annotation.DrawableRes;
-
-import com.novoda.movies.R;
-
-class RateableMovieViewModelFixtures {
```

Comment:
the customisation is in RateableMovieViewModelBuilder - the fixtures (above) just provide some default values so we don't have to keep specifying in the tests about values we don't care about